### PR TITLE
chore: ignore .idea folder for lint

### PR DESCRIPTION
**Pull request by pmuir**
Signed-off-by: Pete Muir <pmuir@bleepbleep.org.uk>

### CODE REVIEW

**Comment by pmuir** [comment]
I was unable to run `npm test` locally without this change.

**Review by matthew-chirgwin — APPROVED** [review]
Looks good, thanks for this! @strimzi/maintainers , could I ask for a second review please?

**Review by scholzj — APPROVED** [review]

**Pull request merged**
